seqr/management/tests/load_rna_seq_tpm_tests.py

```diff
@@ -77,21 +73,26 @@ def test_command(self, mock_open, mock_logger, mock_utils_logger, mock_gzip_open
         self.assertEqual(new_sample.tissue_type, 'WB')
 
         models = RnaSeqTpm.objects.all()
-        self.assertEqual(models.count(), 4)
-        self.assertSetEqual({model.sample for model in models}, {existing_sample, existing_sample2, new_sample})
-        self.assertEqual(models.get(sample=existing_sample, gene_id='ENSG00000240361').tpm, 12.6)
+        self.assertEqual(models.count(), 5)
+        self.assertSetEqual({model.sample for model in models}, set(existing_rna_samples))
+        self.assertEqual(len(models.filter(sample=existing_sample, gene_id='ENSG00000240361')), 0)
         self.assertEqual(models.get(sample=new_sample, gene_id='ENSG00000233750').tpm, 6.04)
 
         mock_logger.info.assert_has_calls([
-            mock.call('create 2 RnaSeqTpm for NA19675_D2'),
             mock.call('create 1 RnaSeqTpm for NA19678_D1'),
         ])
         mock_utils_logger.warning.assert_has_calls([
+            mock.call('Skipped loading row with mismatched tissue types for sample NA19675_D2: muscle, whole_blood', None),
             mock.call('Skipped loading for the following 2 unmatched samples: NA19677, NA19678', None),
         ])
 
-        # Test fails on mismatched tissue
-        mock_gzip_file.__iter__.return_value.pop()
-        mock_gzip_file.__iter__.return_value[2] = 'NA19678_D1\t1kg project nåme with uniçøde\t\tENSG00000233750\t6.04\tfibroblasts\n'
+        # Test a new sample created for a mismatched tissue
+        mock_gzip_file.__iter__.return_value[2] = 'NA19678_D1\t1kg project nåme with uniçøde\tNA19678\tENSG00000233750\t6.55\tfibroblasts\n'
         call_command('load_rna_seq_tpm', 'new_file.tsv.gz', '--ignore-extra-samples')
-        mock_utils_logger.warning.assert_any_call(EXPECTED_MISMATCHED_TISSUE_WARNING, None)
+        models = RnaSeqTpm.objects.filter(sample__sample_id='NA19678_D1')
+        self.assertEqual(models.count(), 2)
+        self.assertEqual(models.get(gene_id='ENSG00000233750', sample__tissue_type='F').tpm, 6.55)
```

Comment:
lets also add a test here that the 2 samples have the 2 different tissue types we expect